tests/deployments/test_cli.py

```diff
@@ -91,13 +91,3 @@ def test_target_help():
     runner = CliRunner()
     res = runner.invoke(cli.target_help, ["--target", f_target])
     assert "Target help is called" in res.stdout
-
-
-def test_run_local():
```

Comment:
Sorry, I think I got this wrong. This `run_local` doesn't seem like it actually starts a flask server. Can we revert this move?